### PR TITLE
Copy changes to home page

### PR DESCRIPTION
**Pull request by tadhg-ohiggins**
Not everything is /
About us; the ants and birds /
Toil on in the sun.

Changes the home page to be less about eRegs and more about ATF-specific aspects. Also changes the upper-left text on every page from “eRegulations” to “ATF eRegulations”.

### CODE REVIEW

**Comment by tadhg-ohiggins** [comment]
For ease of proofing/seeing the differences, this is what it looks like locally:
image

File: atf_eregs/templates/regulations/generic_universal.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% extends "regulations/generic_universal.html" %}
+
+{% block title %} ATF eRegulations {% endblock %}
+
+
+{% block hero %}
+<div class="hero">
+    <div class="inner-wrap group">
+      <div class="primary">
+        <h2 class="hero-header">ATF Title 27 regulations</h2>
```
Comment (cmc333333) [inline atf_eregs/templates/regulations/generic_universal.html]: Another small one -- should this be capitalized "Regulations" or lower-cased "title"? It looks a little odd.

Comment (tadhg-ohiggins) [reply atf_eregs/templates/regulations/generic_universal.html]: I thought “Title 27” (like all the titles) was a proper noun. (The subheading is in sentence case.)

Comment (cmc333333) [reply atf_eregs/templates/regulations/generic_universal.html]: I think you're right about "Title", it just looks a little off. If it's just me, please ignore!

Comment (tadhg-ohiggins) [reply atf_eregs/templates/regulations/generic_universal.html]: This is one of those situations where it's correct but looks wrong. Changing to title case would look less wrong but then be inconsistent. Not sure what the right call is, but I'll leave as is for now.

**Pull request merged**